tests/passes/guard-canonicalize.expect

```diff
@@ -0,0 +1,44 @@
+import "primitives/std.lib";
```

Comment:
I would have assumed that this file was added in #506 already?

Reply:
Hmm.. I'm not sure why that did not get merged.